src/modules/fancyzones/editor/FancyZonesEditor/MainWindow.xaml

```diff
@@ -178,9 +179,9 @@
     
     <StackPanel>
 
-        <TextBlock Name="dialog_Title" Text="{x:Static props:Resources.Choose_Layout}" Style="{StaticResource titleText}"  />
+        <TextBlock Name="DialogTitle" Text="{x:Static props:Resources.Choose_Layout}" Style="{StaticResource titleText}" />

```

Comment:
super nit: seems like the variable naming is lower camel case for variable names.

Reply:
actually, looking across entire project (fz editor) there is no unique standard for that, so let's leave it for now, until we refactor everything